### PR TITLE
Balder/document warmup and then some

### PR DESCRIPTION
**Pull request by baldersheim**
@geirst and @kkraune and @jobergum PR

### CODE REVIEW

File: documentation/content/setup-proton-tuning.html [hunk 1]
```diff
@@ -409,6 +422,21 @@ <h2 id="index">index</h2>
...
+    <ul>
+        <li id="index-warmup-time"><code>time</code>:
+        Specifies in seconds how long the index shall be warmed up before being switch in for serving.
+        During warmup it will receive queries and posting lists will be iterated, but results ignored
+        as they are duplicates of the live index. This will pull in the most important ones in the cache.
+        However as warming up an index will occupy more memory do not turn it on unless you suspect you need it.
+        And always benchmark to see if it is worth it.
+        </li>
+        <li id="index-warmup-unpack"><code>unpack</code>:
+        Controls wether all posting features are pulled in to the cache, or only the most important.
```
Comment (geirst) [inline documentation/content/setup-proton-tuning.html]: Typo: wether -> whether.

File: documentation/content/setup-proton-tuning.html [hunk 2]
```diff
@@ -452,6 +484,41 @@ <h2 id="attribute">attribute</h2>
...
+    </tr><tr>
+        <td>Required:</td><td>No</td>
+    </tr>
+</table>
+Tune various aspect of the db of removed documents. Optional sub-elements:
+<ul>
+    <li id="removed-db-prune"><code>prune</code>
+    <ul>
+        <li id="removed-db-prune-age"><code>age</code>:
+        Specifes how long (in seconds) we must remeber removed documents before we can prune them away.
```
Comment (geirst) [inline documentation/content/setup-proton-tuning.html]: Typo: remeber -> remember

File: documentation/content/setup-proton-tuning.html [hunk 3]
```diff
@@ -452,6 +484,41 @@ <h2 id="attribute">attribute</h2>
...
+        Default is 2 weeks. This sets the upper limit on long a node can be down and still be accepted back
+        in the system without having the index wiped. There is no point in having this any higher than the age of
+        your documents. If you refeed your corpus every day there is no point in having this longer than 24 hours.
+        </li>
+        <li id="removed-db-prune-age"><code>age</code>:
```
Comment (geirst) [inline documentation/content/setup-proton-tuning.html]: Should this be 'removed-db-prune-interval' and 'interval'?

File: documentation/content/setup-proton-tuning.html [hunk 4]
```diff
@@ -452,6 +484,41 @@ <h2 id="attribute">attribute</h2>
...
+        Specifies how often to prune old documents. Default is 600 which is 10 minutes. No need to change default.
```
Comment (geirst) [inline documentation/content/setup-proton-tuning.html]: Add '(in seconds)' after 'often'?

**Comment by baldersheim** [comment]
@geirst Now it should be resolved.

**Pull request merged**